lib/osf-components/addon/components/contributors/user-search/widget/template.hbs

```diff
@@ -0,0 +1,46 @@
+<div ...attributes>
+    <div local-class='InputContainer'>
+        <Input
+            local-class='UserSearchInput'
+            data-test-user-search-input
+            @type='text'
+            @placeholder={{t 'registries.registration_metadata.add_contributors.search_placeholder'}}
+            @value={{this.query}}
+            @enter={{perform this.fetchUsers}}
+        />
+        <Button
+            data-test-user-search-button
+            local-class='ClearButton'
+            @type='create'
+            {{on 'click' (fn (perform this.fetchUsers))}}
+        >
+            {{t 'registries.registration_metadata.add_contributors.search'}}
+        </Button>
+    </div>
+    <Contributors::AddUnregisteredModal
+        @addUnregisteredContributor={{@manager.addUnregisteredContributor}}
+    />
+    <div
+        local-class='UserSearchResultsContainer'
+        data-test-user-search-results
+    >
+        <div local-class='Heading'>
+            <span data-test-heading-name>
+                {{t 'registries.registration_metadata.add_contributors.results_heading'}}
+            </span>
+            <Button
+                local-class='ClearButton'
+                data-analytics-name='Add unregistered contributor'
+                data-test-add-unregistered-contributor
+                @type='create'
+                {{on 'click' @toggleAddContributorWidget}}
```

Comment:
This is the "Done" button shown in the screenshot below. So it toggles off the add contributor widget. However the data analytics name and test selector is confusing because I copy pasted from somewhere else. I'll fix that.
![Screenshot from 2021-02-09 10-21-26](https://user-images.githubusercontent.com/1566880/107385445-e436e500-6ac0-11eb-9233-579c3ee33758.png)